serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -65,10 +65,13 @@ RUN apt-get update && apt-get install -y wget unzip openmpi-bin libopenmpi-dev l
     pip3 cache purge && \
     apt-get clean -y && rm -rf /var/lib/apt/lists/*
 
-# Install PyTorch and TensorRT
+# Install PyTorch
 RUN pip install torch==${TORCH_VERSION} transformers==${transformers_version} accelerate==${accelerate_version} ${peft_wheel} sentencepiece \
```

Comment:
Should torch on a single layer, torch is more than 1 G

Reply:
no, some of those library are trying to install a different version of torch. This is to keep them consistent